sammba/registration/struct.py

```diff
@@ -777,15 +777,18 @@ def anats_to_template(anat_filenames, head_template_filename, write_dir,
 
     if dilated_head_mask_filename is None:
         out_clip_level = clip_level(in_file=head_template_filename)
-        out_threshold = threshold(in_file=head_template_filename,
-                                  thresh=out_clip_level.outputs.clip_val)
-        out_mask_tool = mask_tool(in_file=out_threshold.outputs.out_file,
+        out_calc_threshold = calc(
+            in_file_a=head_template_filename,
+            in_file_b=head_template_filename,
+            expr='ispositive(a-{0}+1)*b'.format(out_clip_level.outputs.clip_val),
```

Comment:
Why +1 after the subtraction?

Reply:
well, because with FSL the thresholding value is included, while the expression `ispositive` means strictly positive in AFNI from what I experimented, and I wanted to have exactly the same output !
Anyway I think it doesn't change a lot, so I will remove this +1 if you agree